### DISCUSSION TITLE
Will IDL4 to Java be considered in future version?

### DISCUSSION THREAD

**Discussion by yuan-ge** [OpenDDS/OpenDDS#4826] [Ideas]
I'm using OpenDDS with Java, and I find the sequence type to Java is array in IDL to java 1.3. It seems that in IDL4, the mapping will be List in Java. It's a great progress!

**Comment by mitza-oci** [comment]
Yes, certainly it will be considered.  Whether or not it will be implemented depends on what the users need and what they want to do in terms of supporting the project.  

The original Java bindings work in OpenDDS was sponsored by an organization that needed to build DDS applications in Java and funded the company I work for, Object Computing, Inc. (OCI) to improve the project.  I don't know for sure, but I'd guess that if a new Java mapping were to be implemented, someone would need to contribute the code or funding to make it happen.
